### ISSUE TITLE
Deploy CLI fails with the error Bad Request: A draft must be in the ‘built’ state before it can be deployed.

### ISSUE THREAD

**Issue by ardenercelik** [auth0/auth0-deploy-cli#395]
The CLI fails unpredictably with the error "Bad Request: A draft must be in the ‘built’ state before it can be deployed.".  If we try to run the CLI again without changing the configuration, deployment passes without any problem. Sometimes deployment of the previously failed action passes, but action with a similar configuration fails to deploy.

The status of the action on Auth0 is deployed but when I run the export it exports the action status as built.

![image](https://user-images.githubusercontent.com/58525494/147647971-6314be08-6845-4f1a-a1d5-65e90af7b87b.png)

YAML definition of the action
```
- name: action-name
    code: ./path/to/action
    dependencies:
      - name: axios
        version: 0.21.4
      - name: retry-axios
        version: 2.6.0
    deployed: true
    runtime: node16
    secrets:
      - name: authzEndpoint
      - name: authzApiKeyId
      - name: authzApiKeySecret
    status: built
    supported_triggers:
      - id: post-login
        version: v2
```

Logs
```
2021-12-27T14:36:28.647Z - error: Problem running command import during stage processChanges when processing type actions
2021-12-27T14:36:28.648Z - error: Problem Deploying actions {"id":"xxxx","name":"action-name"}
Bad Request: A draft must be in the 'built' state before it can be deployed.
```

### Environment
- **Version of this library used: Auth0-deploy-cli version: 7.3.2
- **Version of the platform or framework used, if applicable: Node 16 LTS
- **Other relevant versions (language, server software, OS, browser): Linux
- **Other modules/plugins/libraries that might be involved: axios, axios-retry

**Comment by geoffatsource** [comment]
Same thing happening here

**Comment by geoffatsource** [comment]
👋  this is a big issue, most likely a race-condition with deployment steps. Can someone at Auth0 please respond?

**Comment by shushen** [comment]
Thanks for the report. We've identified this as an issue in our API service and will provide a fix. I'm closing this issue for `deploy-cli` as the tool is behaving as expected.

**Comment by aonamrata** [comment]
> We've identified this as an issue in our API service and will provide a fix

Hi @shushen , Not to repeat the same thing but do you know the version number that has the fix? We are on ` "auth0-deploy-cli": "7.3.1",` and i can update it to latest but the change log does not indicate this issue.

**Comment by heidemn-faro** [comment]
Related: https://community.auth0.com/t/auth0-deploy-cli-v7-3-1-bad-request-a-draft-must-be-in-the-built-state-before-it-can-be-deployed/72667
